Shared/zh-Hans.lproj/Localizable.strings

```diff
@@ -138,6 +138,15 @@
 
 "SAVE_TO_PHOTOS" = "保存到照片";
 
+// History
+"HISTORY" = "历史";
+"%@_AGO" = "%@ ago"; // x 天前
```

Comment:
You're translating the comment and not the actual string here.

Reply:
OK, thanks for pointing that out.